GitHubCard/index.js

```diff
@@ -53,3 +51,105 @@ const followersArray = [];
   luishrd
   bigknell
 */
+
+// Every time you do a .get(response), you must do a .then and .catch to get axios work.
+// `${}` names of list username's
+axios.get('https://api.github.com/users/spettigrew')
+.then(response => {
+  console.log(response)
+  document.querySelector('.cards').appendChild(Cards(response.data)) 
+})
+.catch((error) => {
+  console.log(error)
+})
+
+
+const container = document.querySelector('.container')
+// (image, name, username, location, github, followers, following, bio) instead of (array)
+function Cards (array) {
+let card = document.createElement('div')
+card.classList.add('.card')
```

Comment:
When using classList.add() the thing in quotes does not need the dot and it is messing up your styling right now